### PR TITLE
docs(web links): updated web links pointing to digitaldesign -> tegel

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Updated links pointing to digitaldesign.scania.com to tegel.scania.com

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1750

**How to test**  
1. Check out branch
2. Search for digitaldesign in repo
3. Make sure all hits are only cdn links.

### CODE REVIEW

File: sandbox/vue/README.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ See the running example on [this link](https://scania.github.io/corporate-ui-vue
 
 ## Scania Digital Design System
 
-Visit our Design System at [digitaldesign.scania.com](https://digitaldesign.scania.com/) for more information
+Visit our Design System at [digitaldesign.scania.com](https://tegel.scania.com/) for more information
 
-For more information about how to use SDDS, go to [digitaldesign.scania.com/getting-started/development](https://digitaldesign.scania.com/getting-started/development)
+For more information about how to use SDDS, go to [digitaldesign.scania.com/getting-started/development](https://tegel.scania.com/getting-started/development)
```
Comment (johan-scania) [inline sandbox/vue/README.md:11]: - [digitaldesign.scania.com/getting-started/development] needs to be updated 
- https://tegel.scania.com/getting-started/development is a broken link

File: sandbox/vue/README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Install `components` and `scania-theme` as devDependencies package from NPM in y
 
 ## Use Scania theme
 
-You can use available CSS variables or classes to style your application. See all tokens in [digitaldesign.scania.com](https://digitaldesign.scania.com/).
+You can use available CSS variables or classes to style your application. See all tokens in [digitaldesign.scania.com](https://tegel.scania.com/).
```
Comment (johan-scania) [inline sandbox/vue/README.md:39]: [digitaldesign.scania.com] needs to be updated

File: theme/core/colour/README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ This package contains both SCSS and CSS files available
 
 ### How to use
 
-See all available tokens in [digitaldesign.scania.com](https://digitaldesign.scania.com/foundation/colours/specification)
+See all available tokens in [digitaldesign.scania.com](https://tegel.scania.com/foundation/colours/specification)
```
Comment (johan-scania) [inline theme/core/colour/README.md:18]: - https://tegel.scania.com/foundation/colours/specification is a broken link
- [digitaldesign.scania.com] needs to be updated

File: theme/core/grid/README.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ This grid package contains both SCSS and CSS files available
 
 ### How to use
 
-See implementation in [Tegel Design System](https://digitaldesign.scania.com/foundation/unit-grid-system)
+See implementation in [Tegel Design System](https://tegel.scania.com/foundation/unit-grid-system)
```
Comment (johan-scania) [inline theme/core/grid/README.md:23]: https://tegel.scania.com/foundation/unit-grid-system is a broken link

File: theme/core/typography/README.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ This package contains both SCSS and CSS files available
 
 ### How to use
 
-See all available tokens in [Tegel Design System](https://digitaldesign.scania.com/foundation/typography)
+See all available tokens in [Tegel Design System](https://tegel.scania.com/foundation/typography)
```
Comment (johan-scania) [inline theme/core/typography/README.md:19]: https://tegel.scania.com/foundation/typography is a broken link

**Review by johan-scania — DISMISSED** [review]
- Some broken links
- Some link texts that needs to be updated

File: sandbox/react/README.md [hunk 6]
```diff
@@ -16,9 +16,9 @@ npm start
 
 ## Scania Digital Design System
 
-Visit our Design System at [Digitaldesign.scania.com](https://digitaldesign.scania.com/) for more information
+Visit our Design System at [Digitaldesign.scania.com](https://tegel.scania.com/) for more information
 
-For more information about how to use SDDS, go to [digitaldesign.scania.com/getting-started/development](https://digitaldesign.scania.com/getting-started/development)
+For more information about how to use SDDS, go to [digitaldesign.scania.com/getting-started/development](https://tegel.scania.com/getting-started/development)
```
[the remark this answers is not in the document]
Comment (alexandereneroth) [reply sandbox/react/README.md:22]: Looks fixed

**Review by alexandereneroth — APPROVED** [review]
LGTM

File: sandbox/react/src/components/home/Home.js [hunk 7]
```diff
@@ -25,7 +25,7 @@ const Home = () => {
         <p className="sdds-paragraph-02">
           Check the{' '}
           <a
-            href="https://digitaldesign.scania.com/getting-started/development/installation#angular"
+            href="https://tegel.scania.com/getting-started/development/installation#angular"
```
Comment (mJarsater) [inline sandbox/react/src/components/home/Home.js]:
```suggestion
            href="https://tegel.scania.com/development/getting-started-development/installation#angular"
```

**Pull request merged**